From f71a97db0c59ca320cf478df07c0b9b73dcb30bc Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 14 Feb 2021 13:56:48 +0100
Subject: [PATCH] Revert "libxml2: allow building with iconv support (fixes
 #14637)"

It is causing issues while building following packages:

- openconnect [1]
- lttng-tools,
- augeas
- hs20
- stoken

[1] https://github.com/openwrt/packages/issues/14734

This reverts commit d18692c92638cdc1e643554699cb21e893abf2e9.
---
 libs/libxml2/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/libs/libxml2/Makefile b/libs/libxml2/Makefile
index a9c9332fe..1430dc8f5 100644
--- a/libs/libxml2/Makefile
+++ b/libs/libxml2/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libxml2
 PKG_VERSION:=2.9.10
-PKG_RELEASE:=3
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://xmlsoft.org/sources/
@@ -27,14 +27,13 @@ PKG_BUILD_PARALLEL:=0
 
 include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
-include $(INCLUDE_DIR)/nls.mk
 
 define Package/libxml2
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=Gnome XML library
   URL:=http://xmlsoft.org/
-  DEPENDS:=+libpthread +zlib $(ICONV_DEPENDS)
+  DEPENDS:=+libpthread +zlib
 endef
 
 define Package/libxml2/description
@@ -81,6 +80,7 @@ CONFIGURE_ARGS += \
 	--with-html \
 	--without-ftp \
 	--without-http \
+	--without-iconv \
 	--without-iso8859x \
 	--without-legacy \
 	--with-output \
-- 
2.25.1

